pages/product_list.html

```diff
@@ -0,0 +1,294 @@
+
+            <div class="col-10 ml-l">
+                <div class="rs-row m-l margin-auto">
+                    
+                    <div class="col-sm-4">
+                        <div class="card card-overlay">
+                            <img src="../assets/img/list_img/1.jpg" class="card-image img-c" alt="Card-Image">
+                            <i class="card-close-icon far fa-times-circle"></i>
+                            <div class="card-header">
+                            <div class="card-title">Astoria Brown Watch</div>
+                            </div>
+                            <div class="card-info">
+                            <p class="card-price">Rs.1800</p>
+                            <p class="mrp-price">Rs.3000</p>
+                            <p class="card-discount">60% off</p>
+                            </div>
+                            <div class="card-footer ">
+                                <button class="btn btn-primary "> <i class="fas  fa-shopping-cart"></i> Add to Cart </button>
+                                <button class="btn btn-primary "> Buy Now </button>
+                            </div>
+                            </div>   
+                    </div>
+
+                    <div class="col-sm-4">
+                        <div class="card card-overlay">
+                            <img src="../assets/img/list_img/5.jpg" class="card-image img-c" alt="Card-Image">
+                            <i class="card-close-icon far fa-times-circle"></i>
+                            <div class="card-header"> 
+                            <div class="card-title">Broadway Watch</div>
+                            </div>
+                            <div class="card-info">
+                            <p class="card-price">Rs.1800</p>
+                            <p class="mrp-price">Rs.3000</p>
+                            <p class="card-discount">60% off</p>
+                            </div>
+                            <div class="card-footer ">
+                                <button class="btn btn-primary "> <i class="fas  fa-shopping-cart"></i> Add to Cart </button>
+                                <button class="btn btn-primary "> Buy Now </button>
+                            </div>
+                            </div>   
+                    </div>
+
+
+                    <div class="col-sm-4">
+                        <div class="card card-overlay">
+                            <img src="../assets/img/list_img/7.jpg" class="card-image img-c" alt="Card-Image">
+                            <i class="card-close-icon far fa-times-circle"></i>
+                            <div class="card-header">
+                            <div class="card-title">Soho Watch</div>
+                            </div>
+                            <div class="card-info">
+                            <p class="card-price">Rs.1800</p>
+                            <p class="mrp-price">Rs.3000</p>
```

Comment:
Use HTMl code for Rupee symbol instead of using Rs. `&#8377;`

Reply:
i changed my rupees to doller